content/docs/learning-guide/tutorials/rendering/vegetation-bending-tutorial.md

```diff
@@ -0,0 +1,854 @@
+
+1. Press the **Home** key on your keyboard. This brings up the toolbar at the top.
+
+1. Select **Atom Tools** > **Pass Viewer**.
+
+1. In the pop-up **PassTree View**, enable **Preview Attachment** and **Show Pass Attachments**.
+
+1. In the **PassTree**, find *MotionVectorPass* > *MeshMotionVectorPass* and select the line with `CameraMotion`. 
+
+1. Ensure you are viewing your tree. 
+   
+   {{< note >}}
+   You may only see black on the bottom left preview, because the motion vectors are small, meaning the tree moves minimally. 
+
+   To better see the motion vectors, you can move the camera around or translate the tree quickly. You can also open `MeshMotionVectorVegetationBending.azsl` and scale `OUT.m_motion` in the pixel shader to ensure that the motion vectors' directions are working properly.
+   {{< /note >}}
+
+This video shows the motion vectors when `OUT.m_motion` is scaled by `10000.0`.
+{{< video src="/images/learning-guide/tutorials/rendering/vegetation-bending-tutorial/motionvectortree.mp4" autoplay="true" loop="true" muted="true" width="100%" info="Three trees in the Editor swaying with detail bending, with the motion vector visualizer on the bottom right indicating the direction of movement." >}}
+
+{{< tip >}}
+The Pass Tree Visualizer tool is also helpful for debugging shaders and passes. It allows you to see the output of certain steps of different passes when you select them in the **PassTree**.
+{{< /tip >}}
+
+## Download the AtomTutorial Gem sample
+Now that you've completed this tutorial, you can compare your results to our working version of the vegetation bending material type in the **AtomTutorials Gem** in the [o3de/sample-code-gems repository](https://github.com/o3de/sample-code-gems). You can either download and place the [final working vegetation bending files](https://github.com/o3de/sample-code-gems/tree/main/atom_gems/AtomTutorials/Assets/VegetationBending) in your project, or you can download the Gem and add it to the engine.
+
+If you'd like to download and enable the **AtomTutorials Gem**, do the following:
+1. Download or clone the [o3de/sample-code-gems repository](https://github.com/o3de/sample-code-gems). 
+
+   {{< note >}}
+   If you followed this tutorial, you already downloaded or cloned this repository, but probably moved the `Object` files out of the repository. You can move the files back or re-download/clone the repository.
+   {{< /note >}}
+
+1. Open `VegetationBending.materialtype` and replace all the instances of `{your-path-to-o3de}` with your absolute path to O3DE. 
+
+1. [Register the **AtomTutorials Gem** to the engine](/docs/user-guide/project-config/register-gems). In the command line interface, `cd` into the engine. Then, run
+
+   ```
+   scripts\o3de register -gp {your-path-to-sample-code-gems}\atom_gems\AtomTutorials -espp {your-project-path}
+   ```
+
+   For example, `scripts\o3de register -gp C:\sample-code-gems\atom_gems\AtomTutorials -espp C:\MyProject`.
```

Comment:
I suggest removing the link because you already provide the command that the user needs to run. The link might confuse readers into thinking they need to complete the steps listed there, even though it's already here. 
````suggestion
1. Register the **AtomTutorials Gem** to your project. In the command line interface, run the following command:

   ```
   cd {path-to-o3de-engine}
   scripts\o3de register -gp {your-path-to-sample-code-gems}\atom_gems\AtomTutorials -espp {your-project-path}
   ```

   For example, with paths:
   ```
   scripts\o3de register -gp C:\sample-code-gems\atom_gems\AtomTutorials -espp C:\MyProject
   ```
````